Pipfile

```diff
@@ -4,16 +4,17 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-jupyterlab= "==4.2.4"
+jupyterlab= "==4.2.5"
 jupyterhub = "==5.1.0"
-oauthenticator = "==16.3.1"
+jupyterhub-nativeauthenticator = "==1.2.0"
+oauthenticator = "==17.0.0"
```

Comment:
Both of these are needed?

Reply:
just `jupyterhub-nativeauthenticator` for now. But I thought it's better to get pipfile fixed in one shot as we might use oauth.

Reply:
I wouldn't add a dependency that's not currently needed

Reply:
👍

Reply:
:+1: